pytket/phir/cli.py

```diff
@@ -7,22 +7,21 @@
 ##############################################################################
 
 # mypy: disable-error-code="misc"
+# ruff: noqa: T201
 
 from argparse import ArgumentParser
 from importlib.metadata import version
 
 from pecos.engines.hybrid_engine import HybridEngine  # type:ignore [import-not-found]
+from pecos.foreign_objects.wasmtime import WasmtimeObj  # type:ignore [import-not-found]
```

Comment:
Just sent a PR to pecos at https://github.com/PECOS-packages/PECOS/pull/37 to avoid these type ignores.